packages/website/pages/blog/subscribe.js

```diff
@@ -0,0 +1,125 @@
+import { useEffect, useState } from 'react'
+import Button from '../../components/button.js'
+import countly from '../../lib/countly.js'
+import { subscribe } from '../../lib/subscribe.js'
+import { useRouter } from 'next/router'
+
+export function getStaticProps() {
+  return {
+    props: {
+      title: 'Subscribe - NFT Storage',
+      description: 'Subcribe to the NFT.Storage blog',
+      navBgColor: 'bg-nsltblue',
+      altLogo: true,
+    },
+  }
+}
+
+/**
+ * Subscribe Page
+ *
+ * @param {import('../../components/types.js').LayoutChildrenProps} props
+ * @returns
+ */
+export default function Subcribe({ user }) {
+  const router = useRouter()
+  const [status, setStatus] = useState('')
+  const [errorMsg, setErrorMsg] = useState('')
+  const [disabled, setDisabled] = useState(false)
+  const [email, setEmail] = useState('')
+
+  useEffect(() => setStatus(''), [email])
+  useEffect(() => {
+    if (status === '') setDisabled(false)
+    setErrorMsg('')
+  }, [status])
+
+  /**
+   * @param {import('react').ChangeEvent<HTMLFormElement>} e
+   */
+  const onSubmit = async (e) => {
+    e.preventDefault()
+    const userMail = user?.email || email
```

Comment:
Should we prefer the email address input by the user? They may not want to use their signup email for email marketing.